test/src/sdk-harness/test_projects/storage_map/src/main.sw

```diff
@@ -102,13 +103,21 @@ abi StorageMapTest {
     3]) -> u64;
 }
 
+fn insert_into_u64_to_bool_map_inner(key: u64, value: bool) {
+    storage.map1.insert(key, value);
+}
+
+fn get_from_u64_to_bool_map_inner(key: u64) -> bool {
+    storage.map1.get(key)
+}
```

Comment:
When using functions outside of the trait I prefer the convention of "underscore then name of the function" e.g. `_insert_into_u64_to_bool_map`